snapcraft/cli/lifecycle.py

```diff
@@ -352,8 +353,9 @@ def clean(ctx, parts, unprime, step, **kwargs):
                 instance.clean(part_names=parts)
         else:
             build_provider_class(project=project, echoer=echo).clean_project()
-            # Clear the prime directory on the host
-            lifecycle.clean(project, parts, steps.PRIME)
+            # Clear the prime directory on the host, unless on Windows
```

Comment:
nit: we try to punctuate our comments (https://github.com/snapcore/snapcraft/blob/master/CODE_STYLE.md#comments)